### ISSUE TITLE
Feature Request: SVG export

### ISSUE THREAD

**Issue by hadisfr** [graphia-app/graphia#14]
Hi!
There is no special tag or issue template so I just put my proposal for a feature here. 😃 
I don't know internals of image rendering of Graphia. But I found SVG export very very useful in graph processing. SVG allows users to do a large range of post-processing activities. I could not change label colour of nodes or add an e.g. white border to labels in the current version of Graphia. Also, there is no label-adjust algorithm, yet. If I could get a SVG export of graph I could do all such things in any vector processing software. So I think adding SVG export support, if it's not so difficult, can postpone many rendering for a while and will be very useful.
Thanks for your grate software. 😃

**Comment by timangus** [comment]
I agree that would be a handy feature and something that has been considered in the past, but it would be a challenge to implement. I'll leave the issue open as a reminder.
